tests/clv/test_plotting.py

```diff
@@ -0,0 +1,50 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import pytest
+
+from pymc_marketing.clv.plotting import plot_customer_exposure
+
+
+@pytest.fixture
```

Comment:
This will avoid recreating the same dataframe for every use. Fine if you are not modifying it

```suggestion
@pytest.fixture(scope="module")
```

Reply:
Changing. Thanks for suggestion